### ISSUE TITLE
Wrong extension name for server-sent-events

### ISSUE THREAD

**Issue by goertzenator** [JonathanLorimer/htmx#13]
The [documentation](https://github.com/bigskysoftware/htmx-extensions/tree/main/src/sse) for the SSE extension indicates that it should be selected with...

`hx-ext="sse"`

However using `hxExtension_ ServerSentEvents` gives...

`hx-ext="server-sent-events"`

I can confirm that "sse" works and "server-sent-events" does not.

Also, additional functions to make this extension useful are:

```
sseConnect_ :: Text -> Attributes
sseConnect_ = makeAttributes "sse-connect"

sseSwap_ :: Text -> Attributes
sseSwap_ = makeAttributes "sse-swap"
```

**Comment by JonathanLorimer** [comment]
@goertzenator Ahhh, you are totally right, sorry about this. Fix incoming.

**Comment by JonathanLorimer** [comment]
@goertzenator fixes released in [htmx@1.0.2](https://hackage.haskell.org/package/htmx) and [htmx-lucid@2.0.1](https://hackage.haskell.org/package/htmx-lucid)
